utilities/transactions/transaction_db_impl.h

```diff
@@ -122,5 +123,39 @@ class TransactionDBImpl : public TransactionDB {
   std::unordered_map<TransactionName, Transaction*> transactions_;
 };
 
+class WriteCommittedTxnDBImpl : public PessimisticTxnDB {
```

Comment:
actually let me apply this comment to the other PR that I have: https://github.com/facebook/rocksdb/pull/2691